ui/round/src/interfaces.ts

```diff
@@ -75,10 +75,7 @@ export interface Tv {
 }
 
 interface CrazyData {
-  pockets: {
-    white: CrazyPocket;
-    black: CrazyPocket;
-  };
+  pockets: CrazyPocket[];
```

Comment:
Oh ... the previous typing was actually incorrect, right?

Reply:
Yeah, the server was returning a list